src/main/java/com/codecool/expertsystem/App.java

```diff
@@ -1,26 +1,36 @@
 package com.codecool.expertsystem;
 
+import com.codecool.expertsystem.view.Display;
 import com.codecool.expertsystem.parsers.FactParser;
 import com.codecool.expertsystem.parsers.RuleParser;
+import com.codecool.expertsystem.error.CarNotInXmlException;
 
 /**
  * Hello world!
  *
  */
```

Comment:
Delete this docstring

Reply:
It will be replaced when docstring branch take turn, but ok, will erase it now